fvm/state/accounts_status.go

```diff
@@ -0,0 +1,51 @@
+package state
+
+import (
+	"encoding/hex"
+
+	"github.com/onflow/flow-go/fvm/errors"
+)
+
+type AccountStatus uint8
+
+const (
+	maskExist  byte = 0b0000_0001
+	maskFrozen byte = 0b1000_0000
```

Comment:
Instead of using enum style I did it mask style, cause if something gets frozen and unfrozen we can maintain future states for accounts, while maintaining frozen state